src/client.ts

```diff
@@ -1050,7 +1041,7 @@ export class StreamChat<
       await this.queryChannels(
         { cid: { $in: cids } } as ChannelFilters<ChannelType, UserType, CommandType>,
         { last_message_at: -1 },
-        { limit: 30, recovery: true, last_message_ids: lastMessageIDs },
+        { limit: 30, recovery: true },
```

Comment:
while we're here, is recovery used ?

Reply:
@thesyncim ? ^^

Reply:
this field was used in the early development stages (even before we went to production), its safe to remove